apps/ledger-live-mobile/src/families/osmosis/TransactionConfirmFields.tsx

```diff
@@ -131,13 +133,17 @@ function OsmosisExtendedAmountField({
 }: FieldProps) {
   const unit = getAccountUnit(account);
 
+  if (!(field as unknown as { value: number | BigNumber }).value) return null;
```

Comment:
why not just change the type of `field` in `FieldProps` ?

Reply:
Because it does not work